OWLTools-Runner/src/main/java/owltools/cli/SolrCommandRunner.java

```diff
@@ -1255,20 +1253,9 @@ private void loadGAFDoc(String url, GafDocument gafdoc, EcoTools eco, TaxonTools
 		loader.setTaxonSubsetName(taxonSubsetName);
 		loader.setEcoSubsetName(ecoSubsetName);
 		loader.setGraph(g);
-		try {
```

Comment:
I'm a little sorry to see some of the debugging errors go, but let's see what it looks like in practice?